test/models/item_test.rb

```diff
@@ -4,7 +4,7 @@ class ItemTest < ActiveSupport::TestCase
   attr_reader :item
 
   def setup
-    @item = Item.create(title: 'espresso')
+    @item = Item.create(title: 'espresso', description: "this is black gold", price: 30000)
```

Comment:
Line is too long. [91/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.